python/feathub/processors/flink/flink_processor.py

```diff
@@ -102,7 +102,14 @@ def __init__(
         except ValueError:
             raise FeathubException("Unsupported deployment mode.")
 
-        if self.deployment_mode == DeploymentMode.SESSION:
+        if self.deployment_mode == DeploymentMode.CLI:
+            self.flink_table_builder = FlinkTableBuilder(
+                self._get_table_env(), self.registry
+            )
+            self.flink_job_submitter: FlinkJobSubmitter = (
```

Comment:
Can we make the code style consistent with other instantiations of `FlinkSessionClusterJobSubmitter` in this file?

Reply:
The type of flink_job_submitter is set explicitly to the base class `FlinkJobSubmitter` so that the type checking won't complain. I also added a comment in the code.